src/app/project-overview/clusters-overview/component.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, Input, OnChanges, OnInit} from '@angular/core';
+import {ExternalCluster, ExternalClusterProvider, ExternalClusterState} from '@shared/entity/external-cluster';
+import {Cluster} from '@shared/entity/cluster';
+import {MatTableDataSource} from '@angular/material/table';
+import {ActivatedRoute, Router} from '@angular/router';
+import {PathParam} from '@core/services/params';
+import _ from 'lodash';
+import {getClusterHealthStatus, HealthStatus} from '@shared/utils/health-status';
+import {Health} from '@shared/entity/health';
+import {ClusterListTab} from '@app/cluster/list/component';
+
+@Component({
+  selector: 'km-clusters-overview',
+  templateUrl: 'template.html',
+  styleUrls: ['style.scss'],
+})
+export class ClustersOverviewComponent implements OnInit, OnChanges {
+  @Input() clusters: Cluster[] = [];
+  @Input() clusterHealth: Health[] = [];
+  @Input() externalClusters: ExternalCluster[] = [];
+  @Input() externalClustersEnabled = false;
+  clusterColumns: string[] = ['status', 'name'];
+  clusterDataSource = new MatTableDataSource<Cluster>();
+  externalClusterColumns: string[] = ['status', 'name'];
+  externalClusterDataSource = new MatTableDataSource<ExternalCluster>();
+  projectID = this._activeRoute.snapshot.paramMap.get(PathParam.ProjectID);
```

Comment:
Ideally, it would be best to get the selected project from the service and update the view when it changes.